frontend/src/metabase/query_builder/components/filters/OperatorSelector.jsx

```diff
@@ -39,6 +41,15 @@ export default class OperatorSelector extends Component<*, Props, State> {
         onOperatorChange: PropTypes.func.isRequired
     };
 
+    expandOperators = () => {
+        this.setState({ expanded: true });
+
+        // HACK: Address Safari rendering bug which causes https://github.com/metabase/metabase/issues/5075
+        setTimeout(() => {
```

Comment:
I think you can pass this as a callback to `setState` (2nd arg) and it will do the same thing.